sdk/automation/Microsoft.Azure.Management.Automation/src/Microsoft.Azure.Management.Automation.csproj

```diff
@@ -6,14 +6,14 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Automation</PackageId>
     <Description>Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.</Description>
-    <Version>3.8.4</Version>
+    <Version>3.8.5</Version>
```

Comment:
The new version contains breaking change, so please upgrade the version to 4.0.0

Reply:
updated the version to 4.0.0